### PR TITLE
add collapsible topology files

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

move

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiunimanager/tiunimanager-install-and-maintain.md [hunk 1]
```diff
@@ -83,7 +83,114 @@ TiUniManager 正常运行需要网络环境提供如下端口配置，管理员
...
+  # data_dir: "/em-data/grafana-4111"
+
+filebeat_servers:
+  - host: {ip}
+  # deploy_dir: "/em-deploy/filebeat-0"
+  # data_dir: "/em-data/filebeat-0"
+```
+
+</details>
+
```
Comment (ran-huang) [inline tiunimanager/tiunimanager-install-and-maintain.md:193]:
```suggestion
```

**Review by qiancai — APPROVED** [review]

**Comment by qiancai** [comment]
Before this PR is merged, https://github.com/pingcap/docs-cn/pull/15215/files and its cherry-pick PRs should be merged first.

**Pull request merged**